packages/trpc/server/routers/viewer/bookings/confirm.handler.ts

```diff
@@ -425,3 +406,44 @@ export const confirmHandler = async ({ ctx, input }: ConfirmOptions) => {
 
   return { message, status };
 };
+
+const checkIfUserIsAuthorizedToConfirmBooking = async ({
+  eventTypeId,
+  loggedInUserId,
+  teamId,
+  bookingUserId,
+}: {
+  eventTypeId: number | null;
+  loggedInUserId: number;
+  teamId?: number | null;
+  bookingUserId: number | null;
+}): Promise<void> => {
+  // Check if the user is the owner of the event type
+  if (bookingUserId === loggedInUserId) return;
+
```

Comment:
I think we should have something for system-wide admins too? System wide admins should always have access to all bookings